daemon/api_notices.go

```diff
@@ -218,6 +230,53 @@ func allowedNoticeTypesForInterface(iface string) []state.NoticeType {
 	return types
 }
 
+func postNotices(c *Command, r *http.Request, user *auth.UserState) Response {
```

Comment:
> As far as I understand, this notice type is something produced from within snapd

Not really, `snap-run-inhibit` indicate that `snap run` couldn't start because of an ongoing refresh. snapd have no way of telling when `snap run` run is called.

Thank you for pointing this out, it means that I should document this better.